api/app/Http/Presenters/EventPresenter.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Presenters;
+
+use App\Contracts\PresenterCollectionInterface;
+use App\Entity\Event;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+final class EventPresenter implements PresenterCollectionInterface
+{
+    private EventTypePresenter $eventTypePresenter;
+
+    public function __construct(
+        EventTypePresenter $eventTypePresenter
+    ) {
+        $this->eventTypePresenter = $eventTypePresenter;
+    }
+
+    public function present(Event $event): array
+    {
+        return [
+            'id' => $event->id,
+            'invitee_name' => $event->invitee_name,
+            'invitee_email' => $event->invitee_email,
+            'invitee_question' => 'No questions',
+            'start_date' => $event->start_date,
+            'location' => $event->location,
+            'timezone' => $event->timezone,
+            'created_at' => Carbon::parse($event->created_at)->format('d.m.Y'),
```

Comment:
The date and time should be passed. You will be able to convert it to the needed format on the client.